doc/6/getting-started/vuejs/standalone/index.md

```diff
@@ -0,0 +1,118 @@
+---
+type: page
+code: false
+title: Standalone
+description: Getting started with Kuzzle and VueJS
+order: 0
+---
+
+
+# Getting Started with Kuzzle and VueJS
+
+This section deals with **Kuzzle** (+ **Javascript SDK**) and **VueJS**. We will create **documents** in Kuzzle and subscribe to [document notifications](/sdk/js/6/essentials/realtime-notifications/#document-messages) to develop a realtime chat.
+
+
+## Requirements
+
+- **Node.js** >= 8.0.0 ([install here](https://nodejs.org/en/download/))
+- **Vue CLI** ([install here](https://cli.vuejs.org/guide/installation.html))
+- **Running Kuzzle Stack** ([instructions here](/core/1/guides/getting-started/running-kuzzle/))
+
+## Prepare your environment
+
+Create your VueJS app with Vue CLI. You'll need to select manually the features,
+just add Babel, Linter and select the default options for the other features. 
+```bash
+vue create kuzzle-playground
+```
+
+Install the kuzzle-sdk: 
+```bash
+cd kuzzle-playground
+npm install kuzzle-sdk
+```
+
+In the _App.vue_ file, you should remove the tag, the import and the component registration of the `HelloWorld` component, we won't use it.
+
+## Instanciating Kuzzle SDK
+
+We have to connect the server so that our client can interact with it.
+
+To do this, we have to create _src/services/kuzzle.js_ file to put our kuzzle instance, a bit like a singleton:
+
+<<< ./snippets/kuzzle.js.snippet[js]
+
+We need to import our Kuzzle SDK instance, so just add the following line in your `<script>` tag in the _App.vue_ file: 
+
+<<< ./snippets/App.vue.snippet:3[js]
+
+Then we will [etablish the connection](/sdk/js/6/core-classes/kuzzle/connect/) to kuzzle and create, if they don't [exists](sdk/js/6/controllers/index/exists/), the [index](sdk/js/6/controllers/index/create/) and [collection](sdk/js/6/controllers/collection/create/) of our chat. 
+Add the following `valid()` method in the export of the `<script>` tag of your _App.vue_ file:
+
+<<< ./snippets/App.vue.snippet:2[js]
+
+## Get the username
+First of all, we need to know the user nickname. Let's start by adding the following html code in the `<template>` tag to allow the client type his nickname:
+
+<<< ./snippets/App.vue.snippet:1[html]
+
+As you can see we'll need two properties: `username` and `validate`.
+
+
+## Display the messages
+
+We'll need some properties to manage our messages. Add the following `data` to your _App.vue_
+
+<<< ./snippets/App.vue.snippet:4[js]
+
+Then, create the following functions to fetch and display the messages: 
+
+<<< ./snippets/App.vue.snippet:5[js]
+
+<<< ./snippets/App.vue.snippet:6[js]
+
+The function `fetch_message()` will [search](/sdk/js/6/controllers/document/search/) for the first hundred newest messages and store them in our array before subscribe to notification about `messages` collection. We called it in the `valid()` function we created before.
+
+<<< ./snippets/App.vue.snippet:7[js]
+
+Now, just add the following html code to display the messages:
```

Comment:
Too many "just" usages: limit it to seemingly complex tasks that can be accomplished very easily by a simple action:

```suggestion
Now, add the following HTML code to display the messages:
```